store/postgres/src/dynds/mod.rs

```diff
@@ -47,7 +47,7 @@ pub(crate) fn revert(
     }
 }
 
-pub(crate) fn update_offchain_status(
+pub(crate) fn update_processed_datasources_status(
```

Comment:
Why? What does "processed" add to this name that it didn't have before?

Reply:
Ah nice catch, this was done because initially the plan was to use done_at for on-chain data sources as well, but midway we decided that is not neccessary, will change it back